randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -844,8 +844,9 @@ Extra - room_id: [26]
   > Door to Security Access (Upper)
       Space Jump
   > Door to Crumble City
-      # TODO: Look into requirements further and possibly add SWJ trick. The requirement of space jump makes the advanced tricks in crumble city invalid in non-entrance rando.
-      Screw Attack and Space Jump
+      All of the following:
```

Comment:
RE removed todo: is it not possible to do this with HJ and WJ shenanigans, or HJ and jump extends?

Reply:
No, it is not. The room is too deep. Space jump or blue doors are required to leave the security room. Screw attack with single wall jumps would be the only other way to reach this, hence why I put a TODO so this does not get forgotten about. Hard requiring space jump to get to this node completely eliminates all the expert and hypermode tricks that we factored into logic for crumble city.

Reply:
> Hard requiring space jump to get to this node completely eliminates all the expert and hypermode tricks that we factored into logic for crumble city.

it does not, due to the fact that DB should assume in most cases that every room is isolated.